### ISSUE TITLE
Integrate full text search into the dropdown search box

### ISSUE THREAD

**Issue by kohsah** [gawati/gawati-portal-ui#61]
The main dropdown search box, does not cover site-wide full text search. 
Full text search was implemented (see https://github.com/gawati/gawati-data/issues/9 ). 

It is currently accessible via a document's search tab. 

![image](https://user-images.githubusercontent.com/5685392/37888587-e1d66a32-30e5-11e8-9206-a809c745b017.png)

We need to enable a global site-wide search.  Needs to be a category below "Themes" called "full text search"  with the first 5 results shown there. 

![image](https://user-images.githubusercontent.com/5685392/37888649-2d8407f0-30e6-11e8-844a-b4496e66054e.png)

**Comment by Arunadevi** [comment]
Is it possible create an accumulator and exit the processing after 5 results are obtained?

**Comment by kohsah** [comment]
@Arunadevi yes that is the approach i believe. We don't want to show more than five results in the dropdown.

**Comment by Arunadevi** [comment]
Updated ` 61-siteac-ft-search` branch on the `gawati-data` gitlab repository.  However, I could not write an accumulator in XQuery!  Returning only the first five results instead.
